browser/src/view/components/Button/Button.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './Button.scss';
+
+export const Button = ({ layout, label, Icon, type }) => {
+  const getButtonClassnameByType = type =>
+    ({
+      [Button.TYPE.PRIMARY]: 'primary',
+      [Button.TYPE.SECONDARY]: 'secondary',
+      [Button.TYPE.DESTRUCTIVE]: 'destructive',
+    }[type] || Button.TYPE.PRIMARY);
+
+  return (
+    <a
+      aria-label={`Navigate to the ${label} page`}
+      className={`button ${getButtonClassnameByType(type)}`}
+    >
```

Comment:
Why've you chosen to use an `a` tag here?

Reply:
I think it was from an earlier comment to have me consider an a tag given the use case in the nav. But maybe I misunderstood what you were meaning by that. I'm wondering now if the button component should remain using a button tag, but then perhaps the navbar component renders not a list of li's but maybe instead <a>'s with button components nested inside?

Reply:
If you want to have a button that handles links AND button-like interactions, you'll need to do a bit more than just have a button or an anchor.